ui/src/components/credentials/IssueCredential.tsx

```diff
@@ -75,7 +88,47 @@ export function IssueCredential() {
   });
   const [isLoading, setIsLoading] = useState<boolean>(false);
 
-  const { schemaID } = useParams();
+  const onChangeDid = (changedValues: unknown) => {
+    const parsedDid = didParser.safeParse(changedValues);
+
+    if (parsedDid.success) {
```

Comment:
If somehow this isn't a DID or is null, then it should be handled as well, not ignored.

Reply:
Why? it's called onChangeDid

Reply:
When other parameters change

Reply:
It' was there in the beginning, maybe with too much abstraction, let me know if you prefer the approach pushed